src/java/arjdbc/jdbc/RubyJdbcConnection.java

```diff
@@ -284,7 +284,7 @@ public static String formatTransactionIsolationLevel(final int level) {
     public static int mapTransactionIsolationLevel(final IRubyObject isolation) {
         final Object isolationString;
         if ( isolation instanceof RubySymbol ) {
-            isolationString = isolation.toString(); // RubySymbol.toString (interned)
```

Comment:
let's not change random lines if there isn't a reason ...
esp. since we might need to merge over all the way from 50-stable

Reply:
makes sense here - would have been nice to have it explained in a sep. commit

Reply:
@kares yeah.  I also commented about separate commits too. As separate no doubt a better explanation would have happened too.

In this particular case in JRuby and here I prefer us to use asJavaString because toString is too varied in how it is used (e.g. one use case is for a debugging string).  asJavaString is more explicit that we want a string form (although this is a very minor change so I would have lived with it as toString).